src/builders.ts

```diff
@@ -377,12 +380,27 @@ export class BomBuilder {
     return component
   }
 
+  URL_QUALIFIERS_TO_DROP = ['download_url', 'vcs_url']
+
+  private filterQualifiers (qualifiers: { [key: string]: string }): {} | undefined {
+    const filteredQualifiers = Object.keys(qualifiers)
+      .filter(key => !this.URL_QUALIFIERS_TO_DROP.includes(key))
+      .reduce((obj, key) => {
+        return Object.assign(obj, { key: qualifiers[key] })
```

Comment:
:x: this will not work.  for `{ key: foobar }` the `key` is a literal for a key here, not the variable. 

you need to use `{ [key]: foobar }`